### ISSUE TITLE
Support for XCTest skipping

### ISSUE THREAD

**Issue by ARSteve** [fastlane-community/trainer#41]
There is support now for skipping tests in XCTest ( https://developer.apple.com/documentation/xctest/methods_for_skipping_tests?language=objc ) 

Would be great to have support for processing this output in trainer.

**Comment by shawnce** [comment]
Yeah it would be great to have it forward along skipped tests into the JUnit output using what support that report format has.

**Comment by shawnce** [comment]
Does interest exist to support this? I have not attempted to understand how to contribute to trainer, hoping someone more in the loop could pick up this issue and attempt to implement support for it.
